mono/metadata/marshal.c

```diff
@@ -335,7 +343,7 @@ mono_marshal_init (void)
 		register_icall (mono_string_to_byvalstr, "mono_string_to_byvalstr", "void ptr ptr int32", FALSE);
 		register_icall (mono_string_to_byvalwstr, "mono_string_to_byvalwstr", "void ptr ptr int32", FALSE);
 		register_icall (g_free, "g_free", "void ptr", FALSE);
-		register_icall (mono_object_isinst_icall, "mono_object_isinst_icall", "object object ptr", FALSE);
+		register_icall_no_wrapper (mono_object_isinst_icall, "mono_object_isinst_icall", "object object ptr");
```

Comment:
ok, but then let's be consistent and use it for `ves_icall_System_Threading_Thread_ResetAbort`, `mono_gchandle_new`, `mono_marshal_ftnptr_eh_callback`, etc. as well